types/es-abstract/es2016.d.ts

```diff
@@ -0,0 +1,33 @@
+import ES2015 = require('./es2015');
+
+interface ES2016 extends ES2015 {
+	/**
+	 * @param x
+	 * @param y
+	 *
+	 * @throws {TypeError} If `x` or `y` is a `number` or they're different types.
+	 */
+	SameValueNonNumber(x: unknown, y: unknown): boolean;
```

Comment:
```suggestion
	SameValueNonNumber(x: object | null | undefined | string | boolean | bigint | symbol, y: object | null | undefined | string | boolean | bigint | symbol): boolean;
```

Reply:
That’s what `unknown` is.

Reply:
no - `unknown` includes `number`. This does not.